packages/open-payments/src/client/incoming-payment.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosInstance } from 'axios'
+import { OpenAPI, HttpMethod } from 'openapi'
+import { IncomingPayment, getPath } from '../types'
+import { GetArgs, get } from './requests'
+
+export const getIncomingPayment = async (
+  axios: AxiosInstance,
+  openApi: OpenAPI,
+  args: GetArgs
+): Promise<IncomingPayment> => {
+  return get(
+    axios,
+    args,
+    openApi.createResponseValidator({
```

Comment:
The response validator library uses Ajv, which recommends instantiating once at initialization (vs once per request here)
https://github.com/kogosoftwarellc/open-api/blob/9b58193f17f69a3f6b018a17c79a8c938b7628a2/packages/openapi-response-validator/index.ts#L79-L85
https://ajv.js.org/guide/managing-schemas.html#compiling-during-initialization
(We should probably document this in our `openapi` library...)

Reply:
Good call, updated to create the validators just once. Will also update the docs.